liberty-maven-plugin/src/it/generate-features-it/src/test/java/net/wasdev/wlp/test/dev/it/GenerateFeaturesRestTest.java

```diff
@@ -115,12 +108,12 @@ public void mp3Test() throws Exception {
 
         // verify that the correct features are in the generated-features.xml
         Set<String> features = readFeatures(newFeatureFile);
-        assertTrue(features.contains("mpRestClient-1.4"));
+        assertTrue(features.contains("mpRestClient-1.3"));
```

Comment:
We probably have to make these same changes in ci.gradle

Reply:
Indeed.